### DISCUSSION TITLE
mongodb installation issue

### DISCUSSION THREAD

**Discussion by divyar176** [Homebrew/discussions#2170] [Getting started]
can someone help out with the below error. I am trying to install mongo db via brew
<img width="540" alt="Screen Shot 2021-09-20 at 6 20 26 PM" src="https://user-images.githubusercontent.com/62485459/134088082-028f3f3d-85fe-42b3-bdb2-e6796adfa06c.png">

**Comment by SMillerDev** [comment]
As mentioned in the other discussion: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/
